README.md

```diff
@@ -77,7 +77,7 @@ The developers review the pull request for technical accuracy and team members r
 
 Both the technical review and peer review processes take place in pull requests in GitHub.
 
-After creating and labeling a pull request as outlined above, the developers review the pull request and add comments regarding technical accuracy. Writers receive a notification that comments have been added via email, and when all comments have been addressed, the developers change the label from **Awaiting technical review** to **Technical review passed**.
+After creating and labeling a pull request as outlined above, the developers review the pull request and add comments regarding technical accuracy. Writers receive a notification that comments have been added via email, and when all comments have been addressed, the developers change the label from **awaiting tech review** to **tech review passed**.
```

Comment:
The process here is that the developer adds the label "tech review complete" (not 'passed') as soon as the review has been performed, regardless of whether there were comments - so that way, we don't have a 2nd developer coming along and potentially duplicating the review. A minor point, but if we're going for accuracy, then it's worth mentioning. :)

Reply:
Got it. Fixed.